src/index.js

```diff
@@ -0,0 +1,102 @@
+const registerEvents = () => {
+  console.log('loaded');
+
+  let currentTemp = 71;
+
+  const resetButton = document.getElementById('cityNameResetButton');
+  resetButton.addEventListener('click', () => {
+    document.getElementById('cityNameInput').value = 'Seattle';
+    updateCityName();
+  });
+
+  const updateCityName = () => {
+    console.log('Updating City name');
+    const inputCity = document.getElementById('cityNameInput');
+    const headerCityNameSpan = document.getElementById('headerCityName');
+    headerCityNameSpan.textContent = inputCity.value;
+  };
+
+  const inputCity = document.getElementById('cityNameInput');
+  inputCity.addEventListener('input', (event) => {
+    console.log('Got an input event');
+    updateCityName();
+  });
+
+  const skySelector = document.getElementById('skySelector');
+  skySelector.addEventListener('change', (event) => {
+    console.log('sky selector changed');
+    console.log(event);
+    console.log(skySelector.value);
+    updateWeatherGarden();
+  });
+
+  const tempUp = document.getElementById('tempUp');
+  tempUp.addEventListener('click', () => {
+    console.log('Temp up clicked');
+    currentTemp++;
+    updateTemperatureValue();
+  });
+
+  const tempDown = document.getElementById('tempDown');
+  tempDown.addEventListener('click', () => {
+    console.log('Temp down clicked');
+    currentTemp--;
+    updateTemperatureValue();
+  });
+
+  const updateTemperatureValue = () => {
+    const tempValue = document.getElementById('tempValue');
+    tempValue.textContent = currentTemp;
+    tempValue.classList = [];
+    tempValue.classList.add(getColorClassForTemp(currentTemp));
+    updateWeatherGarden();
+  };
+
+  const getColorClassForTemp = (temp) => {
+    if (temp <= 49) {
+      return 'teal';
+    } else if (50 <= temp && temp <= 59) {
```

Comment:
Notice that we don't actually need to check the lower bound of the range. If we reached this check, then the prior check for whether the temp was below 49 must have failed, meaning it's at least 50.